Lib/pprint.py

```diff
@@ -342,6 +342,31 @@ def _pprint_mappingproxy(self, object, stream, indent, allowance, context, level
 
     _dispatch[_types.MappingProxyType.__repr__] = _pprint_mappingproxy
 
+    def _pprint_simplenamespace(self, object, stream, indent, allowance, context, level):
+        cls_name = object.__class__.__name__
+        if cls_name == "SimpleNamespace":
```

Comment:
If this check is for the sake subclasses then I'd recommend being explicit about it:
```suggestion
        if type(object) is _types.SimpleNamespace:
```